examples/basic/basic_pysdk_demo_image.ipynb

```diff
@@ -0,0 +1,146 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "07daa3b6",
+   "metadata": {},
+   "source": [
+    "![Degirum banner](https://raw.githubusercontent.com/DeGirum/PySDKExamples/main/images/degirum_banner.png)\n",
+    "## Simple example script illustrating object detection\n",
+    "This notebook is one of the simplest examples of how to use the DeGirum PySDK to do AI inference on a graphical file using an object detection model.\n",
+    "\n",
+    "This script works with the following inference options:\n",
+    "\n",
+    "1. Run inference on the DeGirum Cloud Platform;\n",
+    "2. Run inference on a DeGirum AI Server deployed on the local host or on some computer in your LAN or VPN;\n",
+    "3. Run inference on a DeGirum ORCA accelerator directly installed on your computer.\n",
+    "\n",
+    "To try different options, you need to specify the appropriate `hw_location` option. \n",
+    "\n",
+    "You also need to specify your cloud API access token in `degirum_cloud_token`.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17df0fd4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# make sure degirum-tools package is installed\n",
+    "!pip show degirum-tools || pip install degirum-tools"
```

Comment:
Looks like degirum_tools is gone from here. Should get rid of this.

Reply:
@AlexABolotov Done